openedx/core/djangoapps/appsembler/sites/serializers.py

```diff
@@ -87,7 +87,7 @@ def custom_domain_status(self, obj):
 class OrganizationSerializer(serializers.ModelSerializer):
     class Meta:
         model = Organization
-        fields = ('id', 'name', 'short_name', 'edx_uuid')
+        fields = ('id', 'name', 'short_name')
```

Comment:
I think this change breaks AMC site creation. Let's find a way to keep it working.

Until we migrate all sites out of AMC, we will probably need this field to be there. Consider using a feature flag: zd_helpers etc/etc/etc.

```suggestion
        fields = ('id', 'name', 'short_name')
        if zd_helpers.use_org_models():
            fields += ('edx_uuid')
```